### PR TITLE
📦Fix EX settings page load

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/channels/types/external/type.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def get_configuration_context_dict(self, channel):
         }
 
         content_type = config.get(Channel.CONFIG_CONTENT_TYPE, Channel.CONTENT_TYPE_URLENCODED)
-        context["example_content_type"] = "Content-Type: " + Channel.CONTENT_TYPES[content_type]
+        context["example_content_type"] = "Content-Type: " + Channel.CONTENT_TYPES.get(content_type, content_type)
```
Comment (dodobas) [inline temba/channels/types/external/type.py]: `.get(content_type, content_type)` ? 

if Channel.CONTENT_TYPES does not contain key *content_type* then return that key as default value? is this correct ?

Comment (norkans7) [reply temba/channels/types/external/type.py]: Correct, which matches the way we handle that on courier https://github.com/nyaruka/courier/blob/master/handlers/external/external.go#L285

Comment (norkans7) [reply temba/channels/types/external/type.py]: in that case we want to set the content type header to whatever value(raw value) as is set in the config

Comment (nicpottier) [reply temba/channels/types/external/type.py]: Right, content_type can be one of either a constant of `json`, `urlencoded`, `xml` or a real literal.

**Pull request merged**